packages/_infra/src/config.ts

```diff
@@ -15,7 +15,14 @@ export interface BaseMapsConfig {
 }
 
 export const BaseMapsProdConfig: BaseMapsConfig = {
-  CogBucket: ['linz-basemaps', 'linz-basemaps-vector', `linz-basemaps-staging`, 'linz-workflow-artifacts'],
+  CogBucket: [
+    'linz-basemaps',
+    'linz-basemaps-vector',
+    `linz-basemaps-staging`,
+    `linz-basemaps-scratch`,
+    'linz-workflow-artifacts',
+    'linz-workflows-scratch',
```

Comment:
Why the scratch workflow bucket have `s` after `workflows`, does it not stay the same with artifacts bucket?

Reply:
We renamed the cluster to have a trailing "s" so we updated the bucket name at the same time.